shell/package.json

```diff
@@ -129,10 +129,10 @@
     "ufo": "0.7.11",
     "unfetch": "4.2.0",
     "url-parse": "1.5.10",
-    "vue-router": "4.5.0",
+    "vue-router": "4.5.1",
     "vue-select": "4.0.0-beta.6",
     "vue-server-renderer": "2.7.16",
-    "vue": "^3.2.13",
+    "vue": "3.5.18",
```

Comment:
@aalves08 I have some concerns that this change will be too restrictive for extensions. I'm considering if we need to specify a range for backwards compatibility - `>=3.2.13 <=3.5.18`. 

I think that we need to do some testing here to confirm that this change is safe.